tests/server/utils.py

```diff
@@ -28,11 +32,11 @@ def get_index_client() -> TestClient:
     from optimade.server.routers import index_info, links
 
     # # We need to remove the version prefixes in order to have the tests run correctly.
```

Comment:
```suggestion
    # We need to remove the version prefixes in order to have the tests run correctly.
```

Reply:
We should probably actually delete this comment altogether now!

Reply:
```suggestion
```